### ISSUE TITLE
Plotting limits reset when navigating with left arrow key

### ISSUE THREAD

**Issue by magnunor** [hyperspy/hyperspy#2279]
With a `Signal1D` with navigation dimensions, the x-axis plotting limits are reset when navigating with the left arrow key.

```python
import hyperspy.api as hs
s = hs.datasets.artificial_data.get_core_loss_eels_line_scan_signal()
s.plot()
```

Zoom in on some feature using the matplotlib zoom feature. Using the right arrow key for navigating, the zoom is retained, but if the left arrow key is used, the x-lim is reset. Example:

![zoom_left_right_arrow_bug](https://user-images.githubusercontent.com/1690979/68776396-47249c80-0630-11ea-8698-064b13384fd5.gif)

**Comment by hakonanes** [comment]
I've experienced the same with `Signal2D`.

**Comment by ericpre** [comment]
Duplicate of #1866 and fixed in #1870.
Please the plot preferences to know what modifiers key to use, by default it should "crtl"

I don't remember exactly, but I suspect using arrow keys without the modifiers to navigate would be disable in HyperSpy 2.0.

**Comment by magnunor** [comment]
Nice! I was not aware of that change.

> I don't remember exactly, but I suspect using arrow keys without the modifiers to navigate would be disable in HyperSpy 2.0.

I think that is a good idea, since the current way can be a bit confusing.
